### ISSUE TITLE
Error in Read Method Information.vi

### ISSUE THREAD

**Issue by nristovs** [fisothemes/LabVIEW-TwinCAT-ADS#5]
There is an issue with the following VI when used with TwinCAT 3.1.4024.62. The following properties no longer exist in the TwinCAT.Ads.dll

![image](https://github.com/user-attachments/assets/036a229b-072b-41a3-b78b-b907b10621bb)

**Comment by fisothemes** [comment]
Hi @nristovs  I am struggling to reproduce this issue, can you tell me which version of the TwinCAT.Ads assembly you're using? I have 4024.62 installed. The version I was using for development was TwinCAT.Ads(4.3.32.0). I also tried TwinCAT.Ads(4.4.0) from [nuget](https://www.nuget.org/packages/Beckhoff.TwinCAT.Ads/4.4.32). 

Please tell me which version of LabVIEW and its bitness you're using?

Try deleting the wire reference wire or clicking on the greyed out elements.

![github-issue-5(0)](https://github.com/user-attachments/assets/b5ec9b45-ac76-4c9f-b838-1d802fd32568)

To find out which version you're using, right click on the green wire coming out of `get_item` on the `ReadOnlyMethodParameterCollection` invoke node and create a constant. Then right click on that constant, got to `Select .NET Class` and then `Browse`.

My last resort would be to wipe my whole system and reinstall 4024.62 from scratch as well as LV which I am not inclined to do.

**Comment by nristovs** [comment]
I am currently using 4.3.32.0 as well. I've taken two screenshots to show the version and what properties are available for in the reference. This has happened on two separate installations on two computers. Is there any extra packets that should be installed on the in TwinCAT?
![ReferenceError_Fiso](https://github.com/user-attachments/assets/41a74efb-beb4-4bdc-9e23-36ba02ad02f4)
![ReferenceError_Fiso2](https://github.com/user-attachments/assets/9fa50016-d172-403d-93cb-fd1451e70988)

**Comment by fisothemes** [comment]
Interesting. Looks like I'll have to do a clean install on a VM. This might take a while.

**Comment by fisothemes** [comment]
Hi, can I also have the LabVIEW version you used as well as its bitness.

**Comment by fisothemes** [comment]
I found out what your issue was. You're using the wrong DLL. See:

![image](https://github.com/user-attachments/assets/13747256-bc82-439b-a065-f50aca80384b)

Beckhoff may have mixed up the dlls. You can ask for a previous version of the XAR/XAE stating that you have a bug with it.

Another way is to read the .NET Assembly help file and use the correct dll:
![image](https://github.com/user-attachments/assets/15230d37-cf97-4363-b2cb-4a0b137326c3)
Found at View >> .NET Assemblies in Memory >> Help

**Comment by fisothemes** [comment]
This issue is caused by Beckhoff installing the wrong `TwinCAT.Ads.dll` during the installation of `TwinCAT 3.1.4024.62`. The version installed is for WinCE, which doesn't support the [`TcRpcLengthIs`](https://infosys.beckhoff.com/english.php?content=../content/1033/tc3_adssamples_net/1046349195.html&id=) attribute. Since LabVIEW doesn't run on Windows CE, attempting to fix this would only mask the underlying issue.

### Recommended Fix:

1. **Contact Beckhoff** and inform them that `TwinCAT 3.1.4024.62` may be installing the WinCE version of `TwinCAT.Ads.dll` into the Global Assembly Cache (GAC). Ask them to verify and address this potential issue.

2. **Manually replace** the `TwinCAT.Ads.dll` in the GAC at:

   ```
   C:\Windows\Microsoft.NET\assembly\GAC_MSIL\TwinCAT.Ads\
   ```

   with the correct DLL for x86 Windows, located at:

   ```
   C:\TwinCAT\AdsApi\.NET\vA.B.CDEFG\
   ```

   Make sure the version number in the GAC matches the one in the `C:\TwinCAT\...` folder.

> *(Optional)* Uninstall TwinCAT and ask Beckhoff for an older version of the XAE/XAR.

> **Note**:  
> For users on version `4026`, the `TwinCAT` folder is located at `C:\ProgramData\Beckhoff\`.

**Comment by fisothemes** [comment]
Fixed on [Pre-release v1.2.6.14](https://github.com/fisothemes/LabVIEW-TwinCAT-ADS/releases/tag/1.2.6.14)

> **Note:** that this is still an issue on Beckhoff's end, I just found an elegant way to mask it.  I still advice you contact their technical support and inform them about the issue.
